tests/suites/test_suite_x509parse.data

```diff
@@ -3190,6 +3190,10 @@ X509 CRT parse Authority Key Id - Wrong KeyId Tag
 
+X509 CRT parse Authority Key Id - Wrong KeyId Tag Length
+depends_on:MBEDTLS_MD_CAN_SHA1:MBEDTLS_RSA_C
+x509_crt_parse_authoritykeyid:"30820214308201B9A00302010202143960EFDE5674DE1F7B761699CF8E5C024E209452300A06082A8648CE3D040302303B310B3009060355040613024E4C3111300F060355040B0C08506F6C617253534C3119301706035504030C10506F6C617253534C2054657374204341301E170D3233303430363037313834315A170D3433303430313037313834315A303B310B3009060355040613024E4C3111300F060355040B0C08506F6C617253534C3119301706035504030C10506F6C617253534C20546573742043413059301306072A8648CE3D020106082A8648CE3D0301070342000437CC56D976091E5A723EC7592DFF206EEE7CF9069174D0AD14B5F768225962924EE500D82311FFEA2FD2345D5D16BD8A88C26B770D55CD8A2A0EFA01C8B4EDFFA3819A308197301D0603551D0E041604145061A58FD407D9D782010CE5657F8C6346A713BE30760603551D23046F306D80FF5061A58FD407D9D782010CE5657F8C6346A713BEA13FA43D303B310B3009060355040613024E4C3111300F060355040B0C08506F6C617253534C3119301706035504030C10506F6C617253534C205465737420434182143960EFDE5674DE1F7B761699CF8E5C024E209452300A06082A8648CE3D04030203490030460221008F92866DDCE7213AE8F3391759C34A6D731E01C78F9A6BEA3762B2F74C79F627022100D0938EC121E6F6A4B890C4F910A8DEF7EB666EF312DAA09A2BAFBCF0CC03237F":0:"":0:MBEDTLS_ERR_X509_INVALID_EXTENSIONS+MBEDTLS_ERR_ASN1_INVALID_LENGTH
```

Comment:
The test case is failing in some configurations. Looks like a missing dependency.

Reply:
Apart from the dependencies, I'm happy that this is a non-regression test for the change in "Fix parsing of KeyIdentifier (tag length error case)". I have not verified that the change is correct, just that the library change is consistent with the test change.

Reply:
Ups.. I created certificate signed using EC key and this is the reason of failure. Other certificates are using RSA keys, so I force pushed fixed RSA signed cert.